app/services/admin/action_policy_adapter.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Admin
+  # Provides authorization for ActiveAdmin via ActionPolicy
+  #
+  # Dashboard is available to DLSS or Report user or Registrar.
+  #
+  # Reader: all actions available to DLSS.
+  # Report: read available to DLSS or Report user. create available to DLSS.
+  # Submission: index available to DLSS or Report user, show available to DLSS. DLSS can edit once
+  #             submitted. No one can create or delete.
+  class ActionPolicyAdapter < ActiveAdmin::AuthorizationAdapter
+    def authorized?(action, subject = nil)
+      target = policy_target(subject)
+      policy = ActionPolicy.lookup(target, namespace: Admin, default: AdminPolicy)
+      action = format_action(action, subject)
+      policy.new(target, user:).apply(action)
+    end
+
+    private
+
+    def format_action(action, subject)
+      case action
+      when ActiveAdmin::Auth::CREATE
+        :create?
+      when ActiveAdmin::Auth::UPDATE
+        :update?
+      when ActiveAdmin::Auth::READ
+        subject.is_a?(Class) ? :index? : :show?
+      when ActiveAdmin::Auth::DESTROY
+        subject.is_a?(Class) ? :destroy_all? : :destroy?
+      else
+        :"#{action}?"
+      end
```

Comment:
This is where we normalize the activeadmin verbiage with the actionpolicy / rails verbiage.